nixos/tests/mysql/mysql.nix

```
@@ -171,7 +171,7 @@ import ./../make-test-python.nix ({ pkgs, ...} : {
     mariadb.succeed(
         "echo 'use testdb; select test_id from tests;' | sudo -u testuser mysql -u testuser -N | grep 42"
     )
-
+  '' + pkgs.stdenv.lib.optionalString pkgs.stdenv.isx86_64 ''
```

```suggestion
  '' + pkgs.stdenv.lib.optionalString pkgs.stdenv.is64bit ''
```
The test seems to work fine on aarch64

---

Ok done